handler/src/main/java/io/netty/handler/ssl/DefaultOpenSslApplicationProtocolNegotiator.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2014 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.handler.ssl;
+
+import java.util.List;
+
+import static io.netty.util.internal.ObjectUtil.checkNotNull;
+
+/**
+ * OpenSSL {@link ApplicationProtocolNegotiator} for ALPN and NPN.
+ */
+public final class DefaultOpenSslApplicationProtocolNegotiator implements OpenSslApplicationProtocolNegotiator {
```

Comment:
Only thing I wanted to discuss here was the naming convention.  The package uses the convention of `OpenSsl...` or `Jdk...` to clarify context more immediately.  I would like to see consistency in the package one way or the other.